quiz/readme.md

```diff
@@ -13,3 +13,35 @@
 2. Copy `envs/postgres.default.env` to `envs/postgres.env` and fill environmental variables.
 3. Copy `envs/backend.default.env` to `envs/backend.env` and fill environmental variables.
 4. Start services `docker compose -f docker-compose.prod.yaml up --build`.
+
+## Testing
+
```

Comment:
Quick question why not running tests through for e. q. `docker compose exec <service name> pytest`?

Reply:
yeah also valid approach